### ISSUE TITLE
make it possible to target a page by filename inside a chm

### ISSUE THREAD

**Issue by lesensei** [sumatrapdfreader/sumatrapdf#3904]
Some context:
KeePass relies on a chm file for its offline help. When a user clicks on a help item in the application, Windows embedded help viewer displays the relevant page. But Windows embedded help viewer is now deprecated and dysfunctional.
Luckily, KeePass allows to override the software used to display the help with [an advanced setting](https://sourceforge.net/p/keepass/feature-requests/2375/). This however relies on the ability of the viewer to open the page through its internal path. For example, when the user clicks the help button for a new entry, keepass will call the command with the following path "/v2/entry.html".

In short, I'd like to directly display the page "its://1/help/v2/entry.html" (based on what sumatrapdf displays when I right-click the page and hit "properties") of [this chm file](https://keepass.cool/extra/KeePass-2.55-Help-French.zip) from the command line.

Thanks for this great software.

**Comment by GitHubRulesOK** [comment]
interesting they use the internal English page names for ease of calling but this would currently work from a look-up of English to French

SumatraPDF-15900-32.exe -named-dest "La boîte" keepass.chm

**Comment by kjk** [comment]
Will be available in pre-release in about 24 hrs.

Note that the link is: `help/v2/entry.html`. the `its://{number}/` prefix is internal to sumatra and needs to be stripped.

You can also use text from bookmarks e.g. `-named-dest "Recherche"`

**Comment by GitHubRulesOK** [comment]
Ok works but had me wondering why other command line variations did not go to bookmarks like "License" (darn British /American :-)

so the enforced.xml configuration needs to include
```
	<Application>
		<HelpUseLocal>true</HelpUseLocal>
                <HelpUrl>cmd://"C:\Path to\SumatraPDF\SumatraPDF.exe" -named-dest "help/{BASE}" "{APPDIR}\KeePass.chm"</HelpUrl>
		<LastUsedFile>
```
